### PR TITLE
Revise README 

### PR DESCRIPTION
**Pull request by Wreulicke**
This PR will revise README.

The example code on README is not working.
Due to followings.

- "remark-github" needs repository in options
  [ref-code](https://github.com/wooorm/remark-github/blob/32ac83e77a8de9d647922c8679431f6665ad8ab7/lib/index.js#L35)
- "rehype2react" nessd createElement in options
- and trivial others...

In my enviroment, following modules.
- remark-github is 6.0.1

### CODE REVIEW

**Comment by wooorm** [comment]
Nice, thanks!

Could you use single quotes? And add spaces after object properties?

**Comment by wooorm** [comment]
Perfect! @rhysd I’m 👍

**Comment by Wreulicke** [comment]
@woorm Thank you, for your review :D

Oops sorry. It's my mistakes.
I reflected your review.

File: readme.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ to add a Table of Contents and to render GitHub mentions (and other
 cool GH features), and to highlight code blocks.
 
 ```js
-var react = require('react');
+var React = require('react');
+var ReactDOM = require('react-dom')
```
Comment (rhysd) [inline readme.md]: Please `;` to the end of line.

File: readme.md [hunk 2]
```diff
@@ -32,10 +33,14 @@ var rehype2react = require('rehype-react');
 var processor = unified()
   .use(markdown)
   .use(toc)
-  .use(github)
+  .use(github, {
+    repository: 'https://github.com/rhysd/rehype-react'
+  })
```
Comment (rhysd) [inline readme.md]: ditto

**Comment by rhysd** [comment]
Thank you for contributing. Please refer my small comments.

**Comment by Wreulicke** [comment]
Thank you for review.
I reflected.

Could you confirm a reply and the matter?
If I mistake, I will fix.

File: readme.md (hunk 2 above)
Comment (rhysd) [inline readme.md]: Ah, sorry, I got it.

**Pull request merged**